spec/features/support/providers/locations/locations_list_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+feature "Viewing a providers courses" do
+  scenario "i can a provider's locations" do
+    given_i_am_authenticated(user: create(:user, :admin))
+    when_i_visit_a_provider_locations_page
+    then_i_should_see_a_list_of_locations
```

Comment:
Is it worth having two sites? Another one with a dash site code and a Missing URN?

Reply:
I think the logic behind that has already been tested elsewhere - this spec is just to assert that the locations shown belong to the correct provider